hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -348,10 +348,10 @@ checkClose ctx openBefore redeemer =
             signature
       CloseUsed{signature, alreadyDecommittedUTxOHash} ->
         traceIfFalse $(errorCode FailedCloseOutdated) $
-          isNothing deltaUTxOHash'
+          deltaUTxOHash' == hashTxOuts mempty
```

Comment:
We _could_ save some cpu units here by putting the precomputed hash constant here.
```suggestion
          deltaUTxOHash' == emptyHash
```

Where `emptyHash` is ideally defined through template haskell `emptyHash = $(pure $ hashTxOuts mempty)` (might not compile as-is)